### DISCUSSION TITLE
Trying to implement Vehicle Abilities

### DISCUSSION THREAD

**Discussion by SlavicBlin** [YimMenu/YimMenu#1491] [Q&A]
I saw this cool feature by TheGreenBandit and wanted to try to implement it into my version of the menu but i keep getting the error :m_ability_flag is not a member of CVehicleModelInfo"  i would assume this is because i dont have the offset for it so if anyone knows the offset / things needed for it or could link a CVehicleModelInfo.HPP with the offsets done that would be Great <3 👍 

https://github.com/YimMenu/YimMenu/pull/1276

**Comment by xiaoxiao921** [comment]
Literally first message of the original pr link to this https://github.com/Yimura/GTAV-Classes/pull/114

**Reply by ClapperJuan** [reply]
Ohhhhhh I aint even gonna lie i did not see that Thank you sooo much

**Reply by SlavicBlin** [reply]
I didnt know how to view the original PR link, but thank you for linking it 👍
